packages/frontend-2/components/viewer/compare-changes/Version.vue

```diff
@@ -4,27 +4,22 @@
       <PreviewImage :preview-url="version.previewUrl" />
     </div>
     <div
-      v-tippy="createdAt"
+      v-tippy="$getFullDate(version.createdAt)"
       class="bg-foundation-focus inline-block rounded-md px-2 text-xs font-bold truncate text-center py-1"
     >
-      {{ timeAgoCreatedAt }}
+      <span>
+        {{ $getTrunicatedRelativeDate(version.createdAt) }}
+      </span>
       <br />
       {{ isNewest ? 'New' : 'Old' }} Version
     </div>
   </div>
 </template>
 <script setup lang="ts">
-import dayjs from 'dayjs'
 import type { ViewerModelVersionCardItemFragment } from '~~/lib/common/generated/gql/graphql'
 
-const props = defineProps<{
+defineProps<{
```

Comment:
It's still often preferable to keep the results in a computed for caching. If you just use the function directly, it's going to be re-invoked every time the template re-renders, even if the date didn't actually change.

Reply:
I was under the assumption generally these components don't re-render unless the information actually is changed, but I'll change it back to computed 👍

Reply:
@Mikehrn Some other info may have updated and triggered a re-render, the date may still be the same. But since you're invoking a function in the rendering pipeline it's going to re-execute every time a re-render occurs.

https://vuejs.org/guide/essentials/computed.html#computed-caching-vs-methods

Reply:
Yeah I understand that, I was just under the impression that if the info of the model changes in any case the date also changes making it a bit redudant. But in this case it's actually a createdAt instead of updatedAt so it will actually always be the original date; in any case I'll change these back to computed 👍

Reply:
Updated them